### DISCUSSION TITLE
Discord server for Flet!!

### DISCUSSION THREAD

**Discussion by pythonhubdev** [flet-dev/flet#882] [General]
Let's make it happen! A dedicated Discord server for the Flet community would be an amazing addition. Not only will it provide a platform for users to ask questions and share their projects, but it will also foster collaboration and innovation within the community. Let's make it the go-to destination for all things Flet!

**Comment by ndonkoHenri** [comment]
It has been present for a long while now: https://discord.gg/8argTcJt
Hop in.
